package.json

```diff
@@ -172,8 +172,8 @@
     "serve-static": "^1.13.2",
     "sitemap": "^3.2.2",
     "source-map-explorer": "^2.5.2",
-    "stylelint": "^10.1.0",
-    "stylelint-config-standard": "^18.3.0",
+    "stylelint": "^13.13",
+    "stylelint-config-standard": "^22.0.0",
```

Comment:
the stylelint change doesn't seem like it has to be in this PR, why did you need to update it? you can make a separate PR or roll it into your big ol updates PR if you want